src/utils/tagger.go

```diff
@@ -1,58 +1,33 @@
 package utils
```

Comment:
Tightly scoping the PR is a good practice. Let us keep the scope of the PR to spotify v2 upgrade, and remove all the tagger related changes. We can take this up in another PR :)

Reply:
What is your suggestion?
Maybe we can use package versioning? 
To be able to pass the version as an argument or something like that

Reply:
Or i misunderstood. Can i split tagger onto 2 revisions?

Reply:
I was suggesting to revert changes to the `tagger.go` file and restore it to its original state. As far as I understand, the changes here have nothing to do with the spotify library upgrade. Correct me if I'm wrong.

Lets not include these changes here as they are not a part of he v2 migration (which is the theme of this change) afaiu. We can have changes to this file in a seperate PR if required.

Reply:
> 

Tested, you are right! Left the tagger as it was

Reply:
> > 
> 
> Tested, you are right! Left the tagger as it was

Check it out https://github.com/BharatKalluri/spotifydl/pull/23/commits/018bb218d02ecead1e96eecf326595c46be288f4